docs-2.0/8.service-tuning/2.graph-modeling.md

```diff
@@ -14,6 +14,10 @@ There is no perfect method to model in Nebula Graph. Graph modeling depends on
 
 Usually, various types of queries are validated in test scenarios to assess the overall capabilities of the system. However, in most production scenarios, there are not many types of frequently used queries. You can optimize the data model based on key queries selected according to the Pareto (80/20) principle.
 
+### Full volume scanning avoidance
```

Comment:
Thank you 
And `Do not full space scan` not volumn

Reply:
> Thank you And `Do not full space scan` not volumn

oUo, got it